docs/architecture-decisions/adr013-use-node-fetch.md

```diff
@@ -12,33 +12,33 @@ support burden of keeping said package up to date.
 
 ## Decision
 
-Backend (node) packages should use the `node-fetch` package for HTTP data
-fetching. Example:
+Newly written backend (Node.js) packages should use the native `fetch` call for HTTP data
```

Comment:
This might be too soon tbh, It's still [marked as experimental](https://nodejs.org/docs/latest-v20.x/api/globals.html#fetch). I also just ran into some issues over in https://github.com/backstage/backstage/pull/22659/commits/e80a7338df1bad049e342b3ac9bb664e7a7ccdf5 where native fetch seemingly doesn't support `signal`.

Reply:
Wow really? It's been there for a while, surprised that a basic cancellation doesn't work as expected. Should we just close this then?

Reply:
Do the contrib changes still apply?

Reply:
so i guess it may still be an improvement